packages/typedoc-plugin-appium/lib/guards.ts

```diff
@@ -296,7 +296,7 @@ export function isCommandMethodDeclarationReflection(
     ? value.type.declaration.getAllSignatures()
     : value.getAllSignatures();
   return Boolean(
-    signatures?.find((sig) => isReferenceType(sig.type) && sig.type.name === 'Promise')
+    signatures?.find((sig) => sig.type instanceof ReferenceType && sig.type.name === 'Promise')
```

Comment:
I think I prefer using `x instanceof T` when the type guard is that simple; this is why I removed `isReferenceType()`

Reply:
yeah makes sense